common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -139,4 +144,32 @@ public static void warmUpLoop() {
   }
 
   private CommonUtils() {} // prevent instantiation
+
+  /**
+   * Get the current user's group list from Unix by running the command 'groups' NOTE. For
+   * non-existing user it will return EMPTY list.
+   *
+   * @param user user name
+   * @return the groups list that the <code>user</code> belongs to. The primary group is returned
+   *         first.
+   * @throws IOException if encounter any error when running the command
+   */
+  public static List<String> getUnixGroups(String user) throws IOException {
```

Comment:
Other than putting this method in `commonUtils`, maybe `PermissionUtils` is more specific?